### PR TITLE
PLS-903 Songs ES template

### PR DESCRIPTION
**Pull request by dkimdkimdkim**
Index:
- [x] title
- [x] release
- [x] genre
- [x] subgenre
- [x] artist
- [x] label
- [x] provider label
- [x] label owner
- [x] parental advisory.

Tested by:
- [x] PUTting mapping
- [x] POSTing example data
- [x] GETting search results
- [x] PUTting to retrieve suggestions

### CODE REVIEW

File: templates/songs_template.json [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+                    "store": "yes"
+                },
+                "title": {
+                    "type": "string",
+                    "index": "analyzed",
+                    "store": "no"
+                },
+                "release_id": {
+                    "type": "string",
+                    "index": "analyzed",
```
Comment (CarlosMecha) [inline templates/songs_template.json]: Ids and literal strings should be `not_analyzed`

File: templates/songs_template.json [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+                    "store": "no"
+                },
+                "assets": {
+                    "type": "string",
+                    "index": "no",
+                    "store": "no"
+                },
+                "track": {
+                    "type": "long",
+                    "index": "no",
```
Comment (CarlosMecha) [inline templates/songs_template.json]: We might need the `track` later. Set it to `not_analyzed` please

File: templates/songs_template.json [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+                    "store": "no"
+                },
+                "song_key": {
+                    "type": "string",
+                    "index": "no",
```
Comment (CarlosMecha) [inline templates/songs_template.json]: Same with the `bpm` and the `song_key`

File: templates/songs_template.json [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+                    "store": "no"
+                },
+                "duration": {
+                    "type": "string",
+                    "index": "no",
+                    "store": "no"
+                },
+                "isrc": {
+                    "type": "string",
+                    "index": "no",
```
Comment (CarlosMecha) [inline templates/songs_template.json]: Set it to `not_analyzed`

File: templates/songs_template.json [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+                    "type": "string",
+                    "index": "not_analyzed",
+                    "store": "no"
+                },
+                "parental_advisory": {
+                    "type": "boolean",
+                    "index": "no",
+                    "store": "no"
+                },
+                "artists": {
```
Comment (CarlosMecha) [inline templates/songs_template.json]: Artists should be `not_analyzed` for ids and `role_name`

Comment (dkimdkimdkim) [inline templates/songs_template.json]: done

File: templates/songs_template.json (hunk 4 above)
Comment (dkimdkimdkim) [inline templates/songs_template.json]: done

File: templates/songs_template.json (hunk 3 above)
Comment (dkimdkimdkim) [inline templates/songs_template.json]: done

File: templates/songs_template.json (hunk 2 above)
Comment (dkimdkimdkim) [inline templates/songs_template.json]: done

File: templates/songs_template.json (hunk 1 above)
Comment (dkimdkimdkim) [inline templates/songs_template.json]: done

**Pull request merged**